src/Metadata/Resource/Factory/XmlResourceMetadataFactory.php

```diff
@@ -83,9 +83,9 @@ private function getMetadata(string $resourceClass) : array
                 }
 
                 return [
-                    (string) $resource['shortName'] ?? null,
-                    (string) $resource['description'] ?? null,
-                    (string) $resource['iri'] ?? null,
+                    (string) $resource['shortName'] ?: null,
```

Comment:
+1, anyway I thought this was tested but I've the feeling it is not and should be